pkg/bindings/containers/types_list_options.go

```diff
@@ -40,7 +37,6 @@ func (o *ListOptions) ToParams() (url.Values, error) {
 		if !o.Changed(fieldName) {
 			continue
 		}
-		fieldName = strings.ToLower(fieldName)
```

Comment:
this is likely your problem. looking into why.

Reply:
Yup, this gets removed when you go generate, I tried to hard code this back in, but it caused other tools to break.
Strange that I can not get this same error to happen with podman-remote ps --last 1.